package.json

```diff
@@ -1637,4 +1652,4 @@
             "pre-commit": "(git secrets --register-aws || (echo 'Please install git-secrets https://github.com/awslabs/git-secrets to check for accidentally commited secrets!' && exit 1)) && git secrets --pre_commit_hook -- \"$@\" && pretty-quick --staged"
         }
     }
-}
```

Comment:
Try to restrict your changes to just the parts of the code we're touching. We don't need to modify the end of the file like this (probably happened when running the format file command).